orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -191,3 +198,25 @@ func getVpnMconfig(gatewayConfig *models.MagmadGatewayConfigs) *mconfig_protos.O
 
 	return ret
 }
+
+func getStateMconfig(net configurator.Network, gwKey string) *mconfig_protos.State {
+	ret := &mconfig_protos.State{
+		SyncInterval: getRandomizedSyncInterval(gwKey, 60),
+	}
+	netStateConfig := net.Configs["state_config"]
+	if netStateConfig != nil {
+		nsConfig := netStateConfig.(*models.NetworkStateConfig)
+		syncInterval := *nsConfig.SyncInterval
+		ret.SyncInterval = getRandomizedSyncInterval(gwKey, syncInterval)
+	}
+	return ret
+}
+
+// getRandomizedSyncInterval returns the interval received from getSyncInterval
+// as seconds and increases it by a random jitter in the range of
+// [0, 0.2 * getSyncInterval()]. Increased sync interval ameliorates the thundering
+// herd effect at the Orc8r.
+func getRandomizedSyncInterval(gwKey string, syncInterval uint32) uint32 {
+	jitter := math.JitterUint32(syncInterval, gwKey, 0.2)
```

Comment:
How this fn works was recently changed on master. It now returns `n + jitter` instead of just `jitter`

Reply:
Also let's make the value .25, such that default of n=3 minutes means the jitter is up to 45s